test/functional/p2p_invalid_messages.py

```diff
@@ -22,7 +22,12 @@
     P2PInterface,
 )
 from test_framework.test_framework import BitcoinTestFramework
-from test_framework.util import assert_equal
+from test_framework.util import (
+    assert_equal,
+    hex_str_to_bytes,
+)
+import struct
+import time
```

Comment:
nit: IIRC, per PEP8, OS-level and stdlib imports go at the top.

Reply:
Done